packages/amplication-server/src/core/subscription/subscription.controller.ts

```diff
@@ -39,6 +45,19 @@ export class SubscriptionController {
           updateStatusDto.id,
           createSubscriptionInput
         );
+        const userId = updateStatusDto.metadata?.userId;
+        if (
+          createSubscriptionInput.plan === EnumSubscriptionPlan.Pro &&
+          userId
+        ) {
+          await this.analyticsService.track({
+            userId: userId,
+            properties: {
+              workspaceId: updateStatusDto.customer.id,
+            },
+            event: EnumEventType.WorkspacePlanUpgradeCompleted,
+          });
+        }
```

Comment:
Don't we have requirements to track the other events like update, expired, downgrade, canceled?

Reply:
nope, but of cause now we can easily add them.